src/metldata/workflow/template_utils.py

```diff
@@ -15,11 +15,45 @@
 
 """Utility functions for handling Jinja2 templates in workflows."""
 
+from collections.abc import Mapping
+from typing import Any
+
 from jinja2.sandbox import ImmutableSandboxedEnvironment
 
-env = ImmutableSandboxedEnvironment()
+# Random string, effectively disabling Jinja2 blocks and comments
+RANDOM_START_END_STRING = "Wk4CPM:"
+env = ImmutableSandboxedEnvironment(
+    block_start_string=RANDOM_START_END_STRING,
+    block_end_string=RANDOM_START_END_STRING,
+    variable_start_string="{{{",
+    variable_end_string="}}}",
+    comment_start_string=RANDOM_START_END_STRING,
+    comment_end_string=RANDOM_START_END_STRING,
```

Comment:
Seems a bit weird that this doesn't complain about reusing the same delimiters for both blocks and comments

Reply:
Fixed.